internal/actions/reparent.go

```diff
@@ -34,6 +34,24 @@ func Reparent(
 	tx meta.WriteTx,
 	opts ReparentOpts,
 ) (*ReparentResult, error) {
+	branchMeta, exist := tx.Branch(opts.Branch)
+	if !exist {
```

Comment:
so this was not working previously if the current branch was not part of any existing stack?